source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.Tests/Infrastructure/Repositories/MeteringPointRepositoryTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Globalization;
+using System.Threading.Tasks;
+using FluentAssertions;
+using GreenEnergyHub.Charges.Domain.Events.Integration;
+using GreenEnergyHub.Charges.Domain.MeteringPoints;
+using GreenEnergyHub.Charges.Infrastructure.Context;
+using GreenEnergyHub.Charges.Infrastructure.Repositories;
+using Microsoft.EntityFrameworkCore;
+using NodaTime;
+using NodaTime.Text;
+using Xunit;
+
+namespace GreenEnergyHub.Charges.Tests.Infrastructure.Repositories
+{
+    public class MeteringPointRepositoryTests
+    {
+        private readonly DbContextOptions<ChargesDatabaseContext> _dbContextOptions =
+            new DbContextOptionsBuilder<ChargesDatabaseContext>()
+                .UseSqlite("Filename=Test.db")
+                .Options;
+
+        [Fact]
+        public async Task StoreMeteringPointAsync_WhenMeteringPointIsCreated_ValidObjectIsRetrievedFromDatabase()
```

Comment:
I don't see the retrieve part as part of the invariant being ensured here. It's just the way that you happen to verify the storing. I suggest something similar to:

```suggestion
        public async Task StoreMeteringPointAsync_WhenMeteringPointIsCreated_StoresMeteringPointInDatabase()
```

Reply:
renamed in 09cfa573ab5099c5213da23988a29fbd4a0bac37